components/build/jvm-build-service/hacbs-jvm-operator-edit.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: hacbs-jvm-operator-edit
```

Comment:
I don't see this used anywhere?

Reply:
this will aggregate to the standard k8s edit role @stuartwdouglas  via the label at https://github.com/redhat-appstudio/infra-deployments/pull/386/files#diff-e76934b9589181277151d4d57b5adda42ffcf8d46f25d9cfed8def692ccbd244R6

then, the `pipeline` SA set up by openshift pipelines will get these permissions